codex/blockexchange/protobuf/message.nim

```diff
@@ -5,6 +5,17 @@
 
 import pkg/libp2p/protobuf/minprotobuf
 
+import ../../units
+
+const
+  DefaultMaxSize = 100.MiBs.uint
+  MaxEntrySize* = DefaultMaxSize
```

Comment:
I would not actually set this to 100mb, the default of 4mb(?) is more than enough for everything else but the manifest.

Reply:
Yes, the default is 4mb. Until we have https://github.com/codex-storage/codex-research/pull/169 I would leave 100mb for the manifest, it would mean that with the current block size we can handle downloading of files up to ~160GB.